src/components/Tipseen/__stories__/tipseen.stories.mdx

```diff
@@ -0,0 +1,228 @@
+              activeStepIndex={2}
+            />
+          }
+        >
+          <div className="monday-style-story-tipseen_container" />
+        </Tipseen>
+      )
+    }}
+  </Story>
+</Canvas>
+
+### Tipseen with image
+<Canvas>
+  <Story name="Tipseen with image">
+    {() => {
+      const content = [
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>
+      ];
+      return (
+        <Tipseen 
+          position={Tipseen.positions.RIGHT}
+          modifiers={modifiers}
+          isCloseButtonOnImage
+          content={
+            <>
+              <TipseenImage
+                className="monday-style-story-tipseen_image"
+                src={picture}
+              />
+              <TipseenWizard 
+                title="This is a title"
+                steps={content}
+                activeStepIndex={2}
+              />
+            </>
+          }
+        >
+          <div className="monday-style-story-tipseen_big-container" />
+        </Tipseen>
+      )
+    }}
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+        <>
+          <div>
+            <div className="monday-style-story-tipseen_column-item">
+              <Icon icon={Board} iconSize="24" /><span>💫 Research tasks</span>
+            </div>
+            <Tipseen
+              modifiers={modifiers}
+              position={Tipseen.positions.RIGHT} 
+              content={
+                <TipseenContent
+                  isDismissHidden={false}
+                  title="Step1 of 7"
+                  isDismissHidden
+                  children="Click on the “Sales” board to get started"
+                />
+              }
+            >
+              <div className="monday-style-story-tipseen_column-item monday-style-story-tipseen_hovered">  
+                <Icon icon={Board} iconSize="24" /><span>Sales Pipeline</span>
+              </div>
+            </Tipseen>
+            <div className="monday-style-story-tipseen_column-item">
+              <Icon icon={BoardPrivate} iconSize="24" /><span>Marketing</span>
+            </div>
+          </div>
+          <div className="monday-style-story-tipseen_inline-spacing" />
+          </>,
+        description:"Use tipseen in order to guide the user through a new feature or a place they’re not familiar with in the system."
+      },
+      negative: {
+        component:
+          <div className="monday-style-story-tipseen_column-wrapper">
+            <div className="monday-style-story-tipseen_column-spacing"></div>
+            <Tipseen
+              modifiers={modifiers}
```

Comment:
please add the following comment above the modifiers prop:
"The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards. Therefore, there is no need to move this prop in your implementations."

Reply:
please add comment written above